build.gradle

```diff
@@ -439,6 +442,10 @@ test {
         includeCategories 'bio.terra.common.category.Unit'
     }
     outputs.upToDateWhen { false }
+    retry {
```

Comment:
Can you wrap the new `retry` blocks  in `if (isCiServer)`? Unless there's a use case for retrying local builds; in my experience the retries just get in the way when testing locally.

Reply:
I was seeing things fail locally, so I added that in and it fixed it.

Reply:
The problem with using retry for local builds is if your test is really failing and you're trying to fix it. Having it retry when the test really is failing just makes it harder to debug the problem. I think the case where someone really wants to retry locally is much less common and could be done using a local change to `build.gradle` instead of checking it in. A user could also set `CI` in the environment when running tests and get the retry behavior that way too.

Reply:
I'm getting occasional `ClassDefNotFound` errors when I run locally, and throwing retries in helps the tests pass consistently.